testing/vulnerable_app/views/sequence.py

```diff
@@ -107,3 +106,51 @@ def get(self, id, user):
         return {
             'id': widget_id,
         }
+
+
+@ns.route('/side-effect-safe/create')
+class CreateWithSideEffectSafe(Resource):
```

Comment:
Is there a reason why we need a second "safe" create? Can we modify the first one so it meets both needs? What makes the first one "not safe"?